src/main/Main.ts

```diff
@@ -1,12 +1,52 @@
-import {app, ipcMain, nativeImage, BrowserWindow, screen} from "electron";
+import {app, nativeImage, BrowserWindow, screen, globalShortcut, Menu} from "electron";
 import {readFileSync} from "fs";
 import {windowBoundsFilePath} from "../utils/Common";
+import {buildMenuTemplate} from "./Menu";
+
+/**
+ * Contains references to all currently open windows.
+ *
+ * @type {Array}
+ */
+let windows: Electron.BrowserWindow[] = [];
```

Comment:
Maybe something like this?
```TypeScript
interface Zipper {
	add(item: any);
	delete(item: any);
	focus(item: any);
}
```
This acually is the first time for me to write typescript, so i would probably need some help here. If we work on the interface together, i can write the code.

Didn't quite get the `<T>` concept yet, but it might help us to get rid of the `any`'s, right?